spark-extension/src/main/scala/org/apache/spark/sql/auron/NativeConverters.scala

```diff
@@ -819,6 +819,8 @@ object NativeConverters extends Logging {
           }
         }
       case e: Expm1 => buildScalarFunction(pb.ScalarFunction.Expm1, e.children, e.dataType)
+      case e: Contains =>
```

Comment:
I found this function is already implemented. Could you help implement the following ones instead: nvl (81), nvl2 (82), rand (83)?

https://github.com/apache/auron/blob/923db4d9d758aa4961f2138c5d8d5b9c45b07ca5/spark-extension/src/main/scala/org/apache/spark/sql/auron/NativeConverters.scala#L908-L914